src/components/graphs/AnnualCycleGraph/AnnualCycleGraph.js

```diff
@@ -75,48 +80,41 @@ export default class AnnualCycleGraph extends React.Component {
       instance: this.instance,  // for debugging
       prevMeta: null,
       prevArea: null,
-      prevDataSpec: null,
-      dataSpec: null,
+      prevDataspec: null,
+      dataspec: undefined,
```

Comment:
You've made `dataspec` all lowercase in this graph, but it remains camelcase in other graph code. I'm worried that's going to trip me up sometime in the future.

Maybe add an issue that we're transitioning toward all-lowercase spelling of `dataspec`, to remind future programmers both 1) to make that change when they happen to be monkeying around with graph files, and 2) as a reminder that this is in process (ie, they should expect to encounter varying capitalization across files)?

Alternately, if it seems quick and easy enough, you could switch all the graphs now.

Reply:
Good catch; an artefact of introducing the new selector.